site/source/tutorials/internationalization.md

```diff
@@ -0,0 +1,504 @@
+---
+layout: false
+title: Internationalization
+overview: It is increasingly rare that applications reach a single audience that lives in a single region and speaks a single language. This article describes the tools Dojo 2 provides to guarantee an application can be adapted to all users, regardless of their language or address.
+---
+
+## Philosophy and Approach
+
+Internationalization, or i18n, is the process of decoupling an application from a particular language or culture, and is a major requirement of most enterprise applications. As such, internationalization is one of Dojo 2's core concerns. `@dojo/i18n`, Dojo 2's internationalization ecosystem, provides everything that is needed to internationalize and localize an application, from locale-specific messaging to date, number, and unit formatting. While `@dojo/i18n` can be used independently from the rest of Dojo 2, most applications will also rely on `@dojo/widget-core` and `@dojo/cli-build-webpack` to simplify internationalization.
+
+As we will see below, the general process for internationalizing a Dojo 2 application can be outlined as follows:
+
+1. Set the application locale.
+2. Load any [CLDR data](https://github.com/unicode-cldr/) required to correctly format values.
+3. Create message translations for all supported locales.
+4. Delegate formatting to the appropriate `@dojo/i18n` methods when rendering any value that may need to be localized.
+5. Update the application's configuration settings to ensure all required data are included in the build.
+
+## Reading and Setting the Application's Locale
+
+While any widget can have its own dedicated locale (see below), every Dojo 2 application has a single root locale, which can be changed at any point. By default, this root locale is set to that of the user's environment, but can be set to a particular locale at any time. All functionality related to reading and setting the application locale is provided by `@dojo/i18n/i18n`.
```

Comment:
perhaps the instances of see below could you intra-tutorial links, @vansimke is that an option? (assuming that's a good idea, it might not be)